DQM/HLTEvF/plugins/BuildFile.xml

```diff
@@ -32,6 +32,7 @@
   <use   name="MagneticField/Records"/>
   <use   name="MagneticField/Engine"/>
   <use   name="TrackingTools/TrajectoryState"/>
+  <use   name="DQMOffline/Trigger"/>
```

Comment:
Linking to this library is causing plugin load problems in the IB
https://cmssdt.cern.ch/SDT/cgi-bin/logreader/slc7_amd64_gcc700/CMSSW_10_3_X_2018-08-24-2300/pyRelValMatrixLogs/run/136.7802_RunHLTPhy2017B_AODextra+RunHLTPhy2017B_AODextra+DQMHLTonAODextra_2017+HARVESTDQMHLTonAOD_2017/step2_RunHLTPhy2017B_AODextra+RunHLTPhy2017B_AODextra+DQMHLTonAODextra_2017+HARVESTDQMHLTonAOD_2017.log#/

The problem is DQMOffline/Trigger should only be a plug-in, not a shared library. 

@kpedro88 @fabiocos